xml/ha_rear.xml

```diff
@@ -461,73 +443,72 @@
    <title>Using an NFS server to store the file backup</title>
-   <para>
-    &rear; can be used in different scenarios. The following example uses
-    an NFS server as storage for the file backup.
-   </para>
-  </example>
-
-  <procedure>
-   <step>
+  <orderedlist>
+   <listitem>
     <para>
      Set up an NFS server with &yast; as described in the
      <link
       xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-nfs.html">
       &admin; for &sls; &productnumber;</link>.
     </para>
-   </step>
-   <step>
+   </listitem>
+   <listitem>
     <para>
      Define the configuration for your NFS server in the
      <filename>/etc/exports</filename> file. Make sure the directory on the
-     NFS server (where you want the backup data to be available), has the
-     right mount options. For example:
-    </para>
-<screen><replaceable>/srv/nfs</replaceable> *([...],rw,no_root_squash,[...])</screen>
-    <para>
-     Replace <filename>/srv/nfs</filename> with the path to your
-     backup data on the NFS server and adjust the mount options. You might
+     NFS server has the right mount options. For example, you might
      need <literal>no_root_squash</literal> as the
      <command>rear mkbackup</command> command runs as &rootuser;.
+     For more information, see <command>man exports</command>.
     </para>
-   </step>
-   <step>
+   </listitem>
+   <listitem>
     <para>
      Adjust the various <varname>BACKUP</varname> parameters in the
      configuration file <filename>/etc/rear/local.conf</filename> to make
      &rear; store the file backup on the respective NFS server. Find
      examples in your installed system under
      <filename>/usr/share/rear/conf/examples/SLE*-example.conf</filename>.
     </para>
-   </step>
-  </procedure>
+   </listitem>
+  </orderedlist>
+  </example>
+
+  <example xml:id="ex-ha-rear-config-sles-btrfs">
+    <title>Backing up Btrfs subvolumes with <command>tar</command></title>
+    <para>
+     Btrfs is set up with subvolumes, but <command>tar</command>
+     creates backups with the <option>--one-file-system</option> option, which excludes subvolumes.
+     Therefore, the subvolumes must be explicitly included in the &rear; configuration.
+    </para>
+    <para>
+     Add this configuration snippet to <filename>/etc/rear/local.conf</filename> and
+    adjust it according to your setup. For more information and additional options, see the
+    example files at <filename>/usr/share/rear/conf/examples/SLE*-btrfs-example.conf</filename>.
+    </para>
+<screen>OUTPUT=ISO
+BACKUP=NETFS
+BACKUP_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
+BACKUP_PROG_INCLUDE=( /root /boot/grub2/i386-pc /tmp /opt /var /boot/grub2/x86_64-efi /srv /usr/local )</screen>
+    <tip>
+      <title>Directories for <literal>BACKUP_PROG_INCLUDE</literal></title>
+      <para>
+        You can find the directories for your specific system by running the following command:
```

Comment:
Strictl speaking this are no normal directories
but mountpoints, `findmnt -h` shows it as
```
Available output columns:
...
      TARGET  mountpoint
```